spec/reek/configuration/app_configuration_spec.rb

```diff
@@ -99,11 +102,10 @@
       let(:source_via) { "#{directory}/dummy.rb" }
 
       let(:configuration_as_hash) do
-        {
-          directory => { TooManyStatements: { max_statements: 8 } },
-          :IrresponsibleModule => { enabled: false },
-          :TooManyStatements => { max_statements: 15 }
-        }
+        { described_class::DIRECTORIES_KEY =>
+            { directory => { TooManyStatements: { max_statements: 8 } } },
+          'IrresponsibleModule' => { enabled: false },
```

Comment:
Is there a particular reason for changing the key from symbol to string here?

(I notice that we may have an inconsistency here, in that symbols work for the smell types, but perhaps not for the other keys).

Reply:
>> Is there a particular reason for changing the key from symbol to string here?

No, good point, changed it back.